frontend/src/components/TimerComponent/PopUp.js

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import Box from "@material-ui/core/Box";
+
+const style = {
+  position: "absolute",
+  bottom: "0%",
+  right: "0%",
+  width: "20%",
+  bgcolor: "red",
+  border: "2px solid #000",
+  boxShadow: 24,
+  p: 4,
+};
+
+function PopUp({ seconds }) {
+  return (
+    <div>
+      {seconds > 15 && (
+        <Box sx={style}>
```

Comment:
Not sure the Box is needed. Can the styles be applied directly to the p tags?

Reply:
This works
`<p style={style}>You have less than 30 seconds left!</p>`

but style has to be adjusted a bit. I don't think `p` is a built-in react style but `padding` seems to replace it.